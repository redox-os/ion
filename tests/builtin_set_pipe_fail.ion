set -e -p
echo 1
if ! $(false | true)
    echo 2
end
false | true
echo 3
