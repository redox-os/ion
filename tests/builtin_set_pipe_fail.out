1
2
ion: pipeline execution error: early exit: pipeline failed with error code 1
